pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -214,10 +218,8 @@ type EndpointIdentity struct {
 // cilium-operator uses the list of nodes in status to reference count
 // users of this identity, and to expire stale usage.
 type CiliumIdentity struct {
-	// +k8s:openapi-gen=false
```

Comment:
might need do similar clean up for `CiliumNode` struct as well.

Reply:
I think this was on accident. But it seems like removing them had no effect, though? I'll remove them all from the CRD types and see what happens.